content/en/observability_pipelines/set_up_pipelines/generate_metrics/google_pubsub.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Generate Metrics for Google Pub/Sub
+disable_toc: false
+---
+
+## Overview
+
+TKTK
+
+{{% observability_pipelines/use_case_images/generate_metrics %}} 
+
+This document walks you through the following steps:
+1. The [prerequisites](#prerequisites) needed to set up Observability Pipelines
+1. [Setting up Observability Pipelines](#set-up-observability-pipelines)
+1. [Sending logs to the Observability Pipelines Worker](#send-logs-to-the-observability-pipelines-worker-over-fluent)
+
+## Prerequisites
+
+{{% observability_pipelines/prerequisites/fluent%}}
```

Comment:
All references to `fluent` shortcodes were fixed in this commit: https://github.com/DataDog/documentation/pull/25475/commits/3473930f8d826671c68ecc1b0d5a4906131514fe